### DISCUSSION TITLE
2007/05/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E5%96%9D%E9%85%92

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#161] [Announcements]
# 2007/05/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E5%96%9D%E9%85%92

中国人都知道美国自由，美国的确实不像中国有那么法律法规限制个人自由，而且国家不会滥用法律惩罚不听领导话的公民。但是，已有的法律规章，在美国执行起来比在中国要严肃和严格的多。

https://blog.qizhen.xyz/2007/05/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E5%96%9D%E9%85%92

**Comment by ruanqizhen** [comment]
之前博客下的留言：


秋
2007年06月12日 10:09 编辑
醉是什么感觉?

回复
永青
2007年05月21日 09:13 编辑
怪不得steven还问我们在中国是不是谁都可以买酒
